CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ We welcome contributions of:
  * **Documentation Improvements**: You can submit improvements to either:
      * Ozone website. Instructions are here: [Modifying the Ozone Website](https://cwiki.apache.org/confluence/display/HADOOP/Modifying+the+Ozone+Website)
      * Developer docs. These are markdown files [checked into the Apache Ozone Source tree](https://github.com/apache/ozone/tree/master/hadoop-hdds/docs/content).
- * The [wiki pages](https://cwiki.apache.org/confluence/display/HADOOP/Ozone+Contributor+Guide): Please contact us at ozone-dev@hadoop.apache.org and we can provide you write access to the wiki.
+ * The [wiki pages](https://cwiki.apache.org/confluence/display/OZONE/Contributing+to+Ozone): Please contact us at ozone-dev@hadoop.apache.org and we can provide you write access to the wiki.
```

Comment:
Can you also update ozone-dev@hadoop.apache.org -> dev@ozone.apache.org

Reply:
Sure.